webapp/packages/plugin-user-profile/src/UserProfileForm/UserInfoPart/UserProfileFormInfoPartBootstrap.ts

```diff
@@ -25,7 +25,7 @@ export class UserProfileFormInfoPartBootstrap extends Bootstrap {
       name: 'plugin_user_profile_info',
       order: 1,
       panel: () => UserProfileFormInfo,
-      stateGetter: props => () => props.formState.dataContext.get(DATA_CONTEXT_USER_PROFILE_FORM_INFO_PART),
+      stateGetter: props => () => props.formState.getPart(DATA_CONTEXT_USER_PROFILE_FORM_INFO_PART),
```

Comment:
should we do same thing with getPart in similar cases all over the app?

Reply:
this is only case by now, but we will rework how we access parts in other places later (probably we will replace data context with .getPart)